Marlin/src/module/temperature.cpp

```diff
@@ -1804,16 +1804,16 @@ void Temperature::init() {
         temp_range[NR].raw_max -= TEMPDIR(NR) * (OVERSAMPLENR); \
     }while(0)
 
-    #if THERMISTOR_HEATER_0
+    #if THERMISTOR_HEATER_0 && THERMISTOR_HEATER_0 != 999 && THERMISTOR_HEATER_0 != 998
       #ifdef HEATER_0_MINTEMP
       _TEMP_MIN_E(0);
       #endif
       #ifdef HEATER_0_MAXTEMP
-        _TEMP_MAX_E(0);
+        _TEMP_MAX_E(0); 
       #endif
     #endif
 
-    #if HAS_MULTI_HOTEND && THERMISTOR_HEATER_1
+    #if HAS_MULTI_HOTEND && THERMISTOR_HEATER_1 && THERMISTOR_HEATER_0 != 999 && THERMISTOR_HEATER_0 != 998
```

Comment:
Copy & paste issue? isn't it THERMISTOR_HEATER_1 (same for others)

Reply:
yip...

Reply:
Don't hate me for all these "pedantic" notes :-)